etc/kayobe/ansible/wazuh-secrets.yml

```diff
@@ -0,0 +1,31 @@
+---
+- hosts: localhost
+  gather_facts: false
+  vars:
+    wazuh_secrets_path: "{{ kayobe_env_config_path }}/wazuh-secrets.yml"
+  tasks:
+    - name: install passlib[bcrypt]
+      pip:
+        name: passlib[bcrypt]
+        virtualenv: "{{ ansible_playbook_python | dirname | dirname }}"
+
+    - name: Include existing secrets if they exist
+      include_vars: "{{ wazuh_secrets_path }}"
+      ignore_errors: true
```

Comment:
Shouldn't be necessary because these variables are in the standard extra vars path.